llama_index/readers/file/base.py

```diff
@@ -1,36 +1,34 @@
 """Simple reader that reads files of different formats from a directory."""
 import logging
-from copy import deepcopy
 from pathlib import Path
-from typing import Callable, Dict, Generator, List, Optional, Union, cast
+from typing import Callable, Dict, Generator, List, Optional, Type
 
 from llama_index.readers.base import BaseReader
-from llama_index.readers.file.base_parser import BaseParser, ImageParserOutput
-from llama_index.readers.file.docs_parser import DocxParser, PDFParser
-from llama_index.readers.file.epub_parser import EpubParser
-from llama_index.readers.file.image_parser import ImageParser
-from llama_index.readers.file.markdown_parser import MarkdownParser
-from llama_index.readers.file.mbox_parser import MboxParser
-from llama_index.readers.file.slides_parser import PptxParser
-from llama_index.readers.file.tabular_parser import PandasCSVParser
-from llama_index.readers.file.video_audio import VideoAudioParser
-from llama_index.readers.file.ipynb_parser import IPYNBParser
-from llama_index.readers.schema.base import Document, ImageDocument
-
-DEFAULT_FILE_EXTRACTOR: Dict[str, BaseParser] = {
-    ".pdf": PDFParser(),
-    ".docx": DocxParser(),
-    ".pptx": PptxParser(),
-    ".jpg": ImageParser(),
-    ".png": ImageParser(),
-    ".jpeg": ImageParser(),
-    ".mp3": VideoAudioParser(),
-    ".mp4": VideoAudioParser(),
-    ".csv": PandasCSVParser(),
-    ".epub": EpubParser(),
-    ".md": MarkdownParser(),
-    ".mbox": MboxParser(),
-    ".ipynb": IPYNBParser(),
+from llama_index.readers.file.docs_parser import DocxReader, PDFReader
```

Comment:
Makes sense to change these names, but I can image some people being confused that the import changed. We should think about maintaining some kind of changelog for user-facing changes like this

Reply:
I think the old parser are not user facing AFAIK. It's imported to build other "Readers"